examples/network/nodeStyles/circularImageWithDashedBorder.html

```diff
@@ -0,0 +1,99 @@
+<!doctype html>
```

Comment:
Since the new setting should apply to all shapes I'd propose the example to be closer to something like the shapes example. We can pick 5 or 6 shapes to demo it.